framework/components/ecl-navigation/ecl-navigation-lists/_navigation.scss

```diff
@@ -72,6 +72,7 @@ $ecl-navigation-border-width: 2px !default;
 // Small
 
 .ecl-navigation-list--small .ecl-navigation-list__link {
+  border-bottom-width: 0;
```

Comment:
Small display issue (user menu were 2 px larger than site switcher because of this border)